formats/cbor/commonMain/src/kotlinx/serialization/cbor/internal/Encoding.kt

```diff
@@ -22,12 +23,14 @@ private const val BEGIN_ARRAY = 0x9f
 private const val BEGIN_MAP = 0xbf
 private const val BREAK = 0xff
 
+private const val ADDITIONAL_INFORMATION_INDEFINITE_LENGTH = 31
```

Comment:
It this preferred as-is _or_ as?:

```suggestion
private const val ADDITIONAL_INFORMATION_INDEFINITE_LENGTH = 0x1f
```

Reply:
Yeah, it makes sense to write it in hex notation